<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptingConsole</class>
 <widget class="QDialog" name="ScriptingConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scripting</string>
  </property>
  <widget class="QPushButton" name="pbEvaluate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>501</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtScript">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>491</width>
     <height>216</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>255</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLoad">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAbort">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbHelp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
